agent/taskresource/volume/dockervolume.go

```diff
@@ -152,16 +155,52 @@ func (cfg *EFSVolumeConfig) Source() string {
 	return cfg.DockerVolumeName
 }
 
+func (cfg *EFSVolumeConfig) GetType() string {
+	return EFSVolumeType
+}
+
+func (cfg *EFSVolumeConfig) GetVolumeId() string {
+	return ""
```

Comment:
I worry about adding something like this when there's an existing ID for EFS which we might want to return 
```
"fileSystemId": "fs-1234"
```

Reply:
I agree. I recall this was sort of the reason why I originally opted to type-cast since not all volumes types will have fields like volume ID, device name, etc.